esp/scm/ws_dfu.ecm

```diff
@@ -707,34 +707,46 @@ ESPresponse [exceptions_inline, nil_remove, http_encode(0)] DFUGetFileMetaDataRe
     int64 TotalResultRows;
 };
 
-
 ESPrequest ListHistoryRequest
 {
-    string name;
+    string Name;
+};
+
+ESPStruct History
+{
+    string Name;
+    string Operation;
+    string Timestamp;
+    string IP;
+    string Path;
+    string Owner;
+    string Workunit;
 };
 
 ESPresponse [exceptions_inline, nil_remove] ListHistoryResponse
 {
-    binary xmlmap;
+    [depr_ver("1.36")] binary xmlmap;
+    ESParray<ESPStruct History> FileHistory;
```

Comment:
It could be, but you should come to a common ground with @rpastrana what you want and record it into the related JIRA.